### DISCUSSION TITLE
Windows Defender não deixa executar Horse LCL

### DISCUSSION THREAD

**Discussion by CarlosHDOliveira** [HashLoad/horse#358] [Q&A]
Olá,
estou tentando executar o sample LCL do projeto no lazarus, 
mas o Windows Defender derruba a execução e apagar o .exe

![image](https://github.com/HashLoad/horse/assets/23508949/231ed466-05b5-4808-b76a-65fa6d830c27)

Alguém sabe o porque disso?, e como resolver?

Deste já agradeço.

**Comment by viniciussanchez** [comment]
Não vi nada ainda referente a isso até o momento. Pergunta no grupo da hashload no telegram se alguém já passou por isso...
